spec/features/etd_creation_spec.rb

```diff
@@ -216,40 +216,32 @@
     expect(apo_element.first('a')[:href]).to end_with('druid:bx911tp9024') # this is hardcoded in hydra_etd app
     status_element = find_table_cell_following(header_text: 'Status')
     expect(status_element).to have_text('v1 Registered')
-    if Settings.folio.enabled
```

Comment:
since I was already making changes right here, I just yanked the folio switch out.